apps/cms/test/helpers_test.exs

```diff
@@ -390,6 +390,16 @@ defmodule CMS.HelpersTest do
 
       assert parsed_map == []
     end
+
+    test "it handles absence of paragraphs json data" do
+      map_data = %{
+        "status" => [%{"value" => true}]
+      }
+
+      parsed_map = parse_paragraphs(map_data)
+
+      assert parsed_map == []
+    end
```

Comment:
Question: quite a few tests for `parse_paragraphs` exist already. Shouldn't this test be evaluating `from_api` instead, since it was the function that was modified in this PR?

Reply:
I think this test is valuable, but I also agree we could use another for `from_api` for this particular struct :)

Reply:
Sorry for the confusion, I didn't mean we should get rid of this test (but perhaps move it to `helpers_test.exs`, where the others live), just add an additional one accounting for the changes you added to `from_api`.